css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
+  color: blue;
+}
+
+p {
+  color: orange;
+}
+
+.bio {
+  color: green;
+}
+
+#timbl {
+  color: red;
+}
+
+#verou {
```

Comment:
These selectors should be in reverse to demonstrate that specificity will override source order.